dataset.py

```diff
@@ -84,18 +87,22 @@ def process_dataset(dataset, batch_size):
     return dataset
 
 
-def get_dataset(batch_size):
+def get_dataset(batch_size, data_dir=".", random_state=None):
```

Comment:
I added the capability of setting a directory to pull the data from here since folder layouts between your and my machine differ.